swaggerpay/payswaggerlatest.json

```diff
@@ -1121,6 +1200,89 @@
         },
         "additionalProperties": false
       },
+      "AMCSPayGetAllPaymentResponseDetails": {
+        "type": "object",
+        "properties": {
+          "customerId": {
+            "type": "integer",
+            "nullable": true,
+            "example": "1"
+          },
+          "customerName": {
+            "type": "string",
+            "nullable": true,
+            "example": "TestUser"
+          },
+          "ARAccountCode": {
+            "type": "string",
+            "nullable": true,
+            "example": "ACC1001"
+          },
+          "currency": {
+            "type": "string",
+            "nullable": true,
+            "example": "EUR"
+          },
+          "amount": {
+            "type": "Decimal",
+            "nullable": true,
+            "example": "10.02"
+          },
+          "paymentType": {
+            "type": "string",
+            "example": "Credit Card",
+            "nullable": false
+          },
+          "paymentTransactionId": {
+            "type": "integer",
+            "nullable": true,
+            "example": "1"
+          },
+          "paymentTransactionGuid": {
+            "type": "GUID",
+            "nullable": false,
+            "example": "17071E01-E9DF-498A-AD9F-AFDB00A203F0"
+          },
+          "transactionStatus": {
+            "type": "string",
+            "example": "Success",
+            "nullable": false
+          },
+          "paymentId": {
+            "type": "integer",
+            "nullable": true,
+            "example": "1"
```

Comment:
I think we can remove the double quote ("") for this type

Reply:
done